### PR TITLE
Basic chapter structure for documentation

### PR DESCRIPTION
**Pull request by chkal**
Adds the basic structure for the top-level chapters of the documentation.

### CODE REVIEW

File: documentation/src/main/asciidoc/_quickstart.adoc [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    SPDX-License-Identifier: Apache-2.0
+
+////
+== Quickstart
+
+This section describes how to set up a Eclipse Krazo.
```
Comment (lefloh) [inline documentation/src/main/asciidoc/_quickstart.adoc]: Think we can drop the `a`.

File: documentation/src/main/asciidoc/_quickstart.adoc [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+=== Basic setup
+
+The required steps to set up Eclipse Krazo depend on your environment. The easiest way is to
```
Comment (lefloh) [inline documentation/src/main/asciidoc/_quickstart.adoc]: depends lacks an `s`

File: documentation/src/main/asciidoc/_quickstart.adoc (hunk 1 above)
Comment (chkal) [reply documentation/src/main/asciidoc/_quickstart.adoc]: Good catch. Thanks.

File: documentation/src/main/asciidoc/_quickstart.adoc (hunk 2 above)
Comment (chkal) [reply documentation/src/main/asciidoc/_quickstart.adoc]: Fixed.

**Pull request merged**